services/httpd/pprof.go

```diff
@@ -123,6 +136,76 @@ func archiveProfiles(w http.ResponseWriter, r *http.Request) {
 		buf.Reset()
 	}
 
+	// Collect and write out the queries.
+	var allQueries = []struct {
+		name string
+		fn   func() ([]*models.Row, error)
+	}{
+		{"shards", h.showShards},
+		{"stats", h.showStats},
+		{"diagnostics", h.showDiagnostics},
+	}
+
+	tabW := tabwriter.NewWriter(&buf, 8, 8, 1, '\t', 0)
+	for _, query := range allQueries {
+		rows, err := query.fn()
+		if err != nil {
+			http.Error(w, err.Error(), http.StatusInternalServerError)
+		}
+
+		for i, row := range rows {
+			var out []byte
+			// Write the columns
+			for _, col := range row.Columns {
+				out = append(out, []byte(col+"\t")...)
```

Comment:
`col` is a string not a `[]byte`. It looks better as far as I can tell when we have a tab between each column in the output (including between each column header).